packages/iota/src/binary/milestoneOptions/milestoneOptions.ts

```diff
@@ -0,0 +1,97 @@
+// Copyright 2020 IOTA Stiftung
```

Comment:
We're already in 2022, also in some other files

Reply:
I will create separate PR to update the Copyright in the entire project.